app/src/main/java/com/bnyro/contacts/db/obj/LocalContact.kt

```diff
@@ -10,5 +10,5 @@ data class LocalContact(
     val id: Long = 0,
     @ColumnInfo val displayName: String? = null,
     @ColumnInfo val firstName: String? = null,
-    @ColumnInfo val surName: String? = null
+    @ColumnInfo val lastName: String? = null
```

Comment:
By changing that field in the database, with the update of the app the old database will probably be broken and the app unusable without deleting the app data. To avoid that, we should either add database migrations (which is some efforts), or just leave the code internal occurences of `surName` `surName`  and only change the strings visible to the user.

Reply:
if you really want to change the field name, i think it is possible to change it in the model class while keeping it as it is in the database. like this ` @ColumnInfo(name = "surName") val lastName: String? = null `

Reply:
True, it would be but I doubt it would really be worth it having a column info for that, in my opinion it's sufficient to only change it at the places where users actually see it.

Reply:
I went back and undid the changes to the "surName" identifier, and it looks like GitHub updated my PR to reflect that. At least, it looks like the changes to "surName" are missing from the PR now. 

Thank you for the feedback! I didn't realize changing that might break things, but it's good to know.

Reply:
Hah! Actually it looks like my VS Code workspace file snuck in there somehow. I'd better delete that.

Reply:
No worries,  I squash merged the PR in order to keep the commit history clean.
Thanks for your contribution :)